### PR TITLE
APP-1657 : [Back-End] Send new message if maker-checker approved/denied

### PR DESCRIPTION
**Pull request by crepache**
[Back-End] Send new message if maker-checker approved/denied

### CODE REVIEW

File: helpdesk-maker-checker-service/src/main/resources/ActionMessage.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<messageML>
+    <div class="entity" data-entity-id="makerchecker">
+        <card class="barStyle">
+            <header>
+                ${entity['makerchecker'].checker.displayName} ${entity['makerchecker'].state} this message. It has been delivered to the client(s).
```
Comment (ardsilva) [inline helpdesk-maker-checker-service/src/main/resources/ActionMessage.xml]: When the state is approved, this phrase It has been delivered to the client(s). are correct, but, if denied, it should be
It has not been delivered to the client(s).

**Review by ardsilva — CHANGES_REQUESTED** [review]
Only one thing to change

File: helpdesk-maker-checker-service/src/main/resources/ActionMessage.xml (hunk 1 above)
Comment (crepache) [reply helpdesk-maker-checker-service/src/main/resources/ActionMessage.xml]: Done.

**Review by ardsilva — APPROVED** [review]
LGTM

**Pull request merged**